src/action-types/index.js

```diff
@@ -0,0 +1,6 @@
+const { makeActionTypes } = require('utils/redux-helpers');
+
+module.exports = makeActionTypes({
+    COUNTER: require('./counter'),
+    MOON_PHASE: require('./moon-phase')
+});
```

Comment:
This file is a pain to me, keep forgetting to update it -- would be nice to think of some wiring to automate this

Reply:
Sounds good, let's think about that later, i.e. if we decide to go with this approach.